Sources/EmbraceConfigInternal/EmbraceConfigurable/EmbraceConfigurable.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Copyright © 2024 Embrace Mobile, Inc. All rights reserved.
+//
+
+import Foundation
+
+/// This protocol is used to add configuration to the runtime of the SDK
+/// It is used to configure the ongoing behavior of the SDK
+@objc public protocol EmbraceConfigurable {
+    var isSDKEnabled: Bool { get }
+
+    var isBackgroundSessionEnabled: Bool { get }
+
+    var isNetworkSpansForwardingEnabled: Bool { get }
+
+    var internalLogLimits: InternalLogLimits { get }
```

Comment:
I wanted to make this interface as concise as possible. Instead of getters for all the severities, I went with exposing the `InternalLogLimits` object directly. 

We may want to rename `InternalLogLimits` to differentiate it from the `Internal` target suffix we use.

Reply:
This config determines if these logs are emitted. I think its ok if a client determines they don't want these logs alongside their data.